src/phases/scale/MixPhasesRateLimited.yml

```diff
@@ -19,6 +19,8 @@ Keywords:
 - update
 - deleteOne
 - delete
+- RateLimited
+- GlobalRate
```

Comment:
Could you lower case these? There's no enforced rule, but I think they have been lower case unless there was a compelling reason to be different. For example deleteOne uses that weird capitalization in the operation spec.

Reply:
Yeah sure will do